frame/nomination-pools/src/lib.rs

```diff
@@ -706,20 +706,24 @@ impl<T: Config> BondedPool<T> {
 			// We checked for zero above
 			.div(bonded_balance);
 
+		let min_points_to_balance = MinPointsToBalance::<T>::get();
+
 		// Pool points can inflate relative to balance, but only if the pool is slashed.
 		// If we cap the ratio of points:balance so one cannot join a pool that has been slashed
-		// 90%,
-		ensure!(
-			points_to_balance_ratio_floor < MinPointsToBalance::<T>::get().into(),
-			Error::<T>::OverflowRisk
-		);
-		// while restricting the balance to 1/10th of max total issuance,
-		let next_bonded_balance = bonded_balance.saturating_add(new_funds);
-		ensure!(
-			next_bonded_balance <
-				BalanceOf::<T>::max_value().div(MinPointsToBalance::<T>::get().into()),
-			Error::<T>::OverflowRisk
-		);
+		// by `min_points_to_balance`%, if not zero.
+
+		if !min_points_to_balance.is_zero() {
```

Comment:
Wondering if this is the best place to check if zero? Can MinPointsToBalance be checked via integrity tests?

Reply:
Ah sorry I saw that incorrectly.  
Yea this is not the best spot: It is written in two places: `GenesisConfig::build` and in `set_configs`.  
So it should be checked twice. You should probably also add it to the `sanity_checks`.  

Also in the `is_zero` case you should return an `OverflowRisk` error, right?

Reply:
Post @kianenigma 's advice that I agree with, `MinPointsToBalance` has been changed to a constant. 

We can now check the value is not zero in `integrity_test()`.